webpack.config.js

```diff
@@ -56,6 +56,7 @@ const CSS_LOADER_CONFIG = [
   {
     loader: 'postcss-loader',
     options: {
+      sourceMap: true,
```

Comment:
basically, this should _not_ produce sourcemaps for `npm run build` or `npm run build:min`